common-npm-packages/docker-common-v2/package.json

```diff
@@ -19,12 +19,13 @@
         "@types/q": "1.5.4",
         "del": "2.2.0",
         "q": "1.4.1",
-        "azure-pipelines-task-lib": "^3.1.0"
+      "azure-pipelines-task-lib": "^3.1.0",
+      "azure-pipelines-tasks-azure-arm-rest-v2": "2.208.0"
     },
     "devDependencies": {
         "typescript": "4.0.2"
     },
-    "scripts": {
-        "build": "cd ../build-scripts && npm install && cd ../docker-common-v2 && node make.js"
-    }
+  "scripts": {
+    "build": "node ../build-scripts/downloadArchive.js https://vstsagenttools.blob.core.windows.net/tools/7zip/1/7zip.zip ./tools && node make.js"
```

Comment:
Can you please educate me why this change was required ?

Reply:
I assume this is for testing and you will revert this before merge ? can you confirm ?

Reply:
I will remove this before merge.